Orange/widgets/utils/tests/test_state_summary.py

```diff
@@ -100,13 +101,17 @@ def make_table(attributes, target=None, metas=None):
 
 
 class TestUtils(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
```

Comment:
Is this ever used?

Reply:
It came up as an error on travis before so I set it, but now it seems fine without it.